index.html

```diff
@@ -73,7 +73,7 @@
         <h3>Security</h3>
 
         <details>
-          <summary>Sandstorm protects you from evil (or buggy) apps while making sure security never gets in the way.</summary>
+          <summary>Sandstorm runs each app in a tight sandbox, keeping you safe from evil (or buggy) apps while making sure security never gets in the way.</summary>
```

Comment:
Another proposal might be to say, "Sandstorm is designed by security wonks" pre-click.